src/main/java/Duke/Commands/Command.java

```diff
@@ -0,0 +1,14 @@
+package Duke.Commands;
+
+import Duke.DukeException.DukeException;
+import Duke.System.Storage;
+import Duke.System.TaskList;
+import Duke.System.Ui;
+
+public abstract class Command {
+    public abstract void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException;
```

Comment:
You could have a constructor here with the inputs being tasks, ui and storage and have the other Commands use this constructor to sort of bring a uniformity in your code.